scripts/src/models/applications.ts

```diff
@@ -99,15 +99,22 @@ export class AppOffer extends BaseEntity {
 				.leftJoinAndSelect("app_offer.app", "app")
 				.where("app_offer.appId = :appId", { appId })
 				.andWhere("offer.type = :type", { type })
-				.orderBy("offer.amount", type === "earn" ? "DESC" : "ASC")
-				.addOrderBy("offer.id", "ASC")
+				.orderBy("offer.ordering", "ASC")
```

Comment:
maybe still leave the other orderBy statements as secondary, so when you deploy to production you can leave `ordering` empty to begin with. In any case add the alter table statement here in the PR so we can review it too

Reply:
What I thought is to alter the table first: add a column, populate it with data and afterwards to deploy this fix
How to add the `alter table statement` code to the PR? By a comment in the code or comment in the PR itself?

Reply:
just comment in the PR

Reply:
Done
I added a unique index to the column for performance and consistency